### PR TITLE
General MuJoCo Env Documention Cleanup

### PR DESCRIPTION
**Pull request by Kallinteris-Andreas**
# Description
add excluded observation in the obs tables
renamed links → body parts
cleaned observation description sizes
fixed some typos
Copied `Humamoid` obs → `HumanoidStandup`

This is probably the last MuJuCo v4 environment doc related PR.
All future doc changes will be in MuJoCo v5 envs

### CODE REVIEW

File: gymnasium/envs/mujoco/walker2d_v4.py [hunk 1]
```diff
@@ -15,134 +15,135 @@
...
-
-
-    ## Version History
-
-    * v4: All MuJoCo environments now use the MuJoCo bindings in mujoco >= 2.1.3
-    * v3: Support for `gymnasium.make` kwargs such as `xml_file`, `ctrl_cost_weight`, `reset_noise_scale`, etc. rgb rendering comes from tracking camera (so agent does not run away from screen)
-    * v2: All continuous control environments now use mujoco-py >= 1.50
-    * v1: max_time_steps raised to 1000 for robot based tasks. Added reward_threshold to environments.
-    * v0: Initial versions release (1.0.0)
+        ## Description
```
Comment (pseudo-rnd-thoughts) [inline gymnasium/envs/mujoco/walker2d_v4.py:18]: Why is this indented compared to before?

File: gymnasium/envs/mujoco/hopper_v4.py [hunk 2]
```diff
@@ -37,31 +37,31 @@ class HopperEnv(MujocoEnv, utils.EzPickle):
...
+    | 1   | angle of the torso                               | -Inf | Inf | rooty                            | hinge | angle (rad)              |
     | 2   | angle of the thigh joint                         | -Inf | Inf | thigh_joint                      | hinge | angle (rad)              |
     | 3   | angle of the leg joint                           | -Inf | Inf | leg_joint                        | hinge | angle (rad)              |
     | 4   | angle of the foot joint                          | -Inf | Inf | foot_joint                       | hinge | angle (rad)              |
-    | 5   | velocity of the x-coordinate of the top          | -Inf | Inf | rootx                            | slide | velocity (m/s)           |
-    | 6   | velocity of the z-coordinate (height) of the top | -Inf | Inf | rootz                            | slide | velocity (m/s)           |
-    | 7   | angular velocity of the angle of the top         | -Inf | Inf | rooty                            | hinge | angular velocity (rad/s) |
+    | 5   | velocity of the x-coordinate of the torso          | -Inf | Inf | rootx                          | slide | velocity (m/s)           |
```
Comment (pseudo-rnd-thoughts) [inline gymnasium/envs/mujoco/hopper_v4.py:62]: For my OCD, could you fix the table column `|` to all line up

File: gymnasium/envs/mujoco/humanoidstandup_v4.py [hunk 3]
```diff
@@ -52,16 +52,23 @@ class HumanoidStandupEnv(MujocoEnv, utils.EzPickle):
     | 16  | Torque applied on the rotor between the left upper arm and left lower arm          | -0.4        | 0.4         | left_elbow                       | hinge | torque (N m) |
 
     ## Observation Space
+    Observations consist of positional values of different body parts of the Humanoid,
+     followed by the velocities of those individual parts (their derivatives) with all the
```
Comment (pseudo-rnd-thoughts) [inline gymnasium/envs/mujoco/humanoidstandup_v4.py:56]: Remove extra space at the beginning of the line

File: gymnasium/envs/mujoco/humanoidstandup_v4.py [hunk 4]
```diff
@@ -52,16 +52,23 @@ class HumanoidStandupEnv(MujocoEnv, utils.EzPickle):
...
+    will be returned in `info` with keys `"x_position"` and `"y_position"`, respectively.
 
-    **Note:** The x- and y-coordinates of the torso are being omitted to produce position-agnostic behavior in policies
-
-    The observation is a `ndarray` with shape `(376,)` where the elements correspond to the following:
+    However, by default, the observation is a `Box(-Inf, Inf, (376,), float64)`. The elements correspond to the following:
 
     | Num | Observation                                                                                                     | Min  | Max | Name (in corresponding XML file) | Joint | Unit                       |
     | --- | --------------------------------------------------------------------------------------------------------------- | ---- | --- | -------------------------------- | ----- | -------------------------- |
+    | excluded | x-coordinate of the torso (centre)                                                                         | -Inf | Inf | root                             | free  | position (m)               |
```
Comment (pseudo-rnd-thoughts) [inline gymnasium/envs/mujoco/humanoidstandup_v4.py:70]: move to the end

File: gymnasium/envs/mujoco/humanoidstandup_v4.py [hunk 5]
```diff
@@ -120,8 +126,55 @@ class HumanoidStandupEnv(MujocoEnv, utils.EzPickle):
...
     - *cfrc_ext:* This is the center of mass based external force on the body.  It has shape
     14 * 6 (*nbody * 6*) and hence adds to another 84 elements in the state space.
-    where *nbody* stands for the number of bodies in the robot and *nv* stands for the number
-    of degrees of freedom (*= dim(qvel)*)
+    where *nbody* stands for the number of bodies in the robot and *nv* stands for the
+    number of degrees of freedom (*= dim(qvel)*)
+
+    The body parts are:
+
+    | id (for `v2`,`v3`,`v4`) | body part |
```
Comment (pseudo-rnd-thoughts) [inline gymnasium/envs/mujoco/humanoidstandup_v4.py:134]: Fix columns

File: gymnasium/envs/mujoco/humanoidstandup_v4.py [hunk 6]
```diff
@@ -120,8 +126,55 @@ class HumanoidStandupEnv(MujocoEnv, utils.EzPickle):
...
+    | 8 | left_sin |
+    | 9 | left_foot |
+    | 10 | right_upper_arm |
+    | 11 | right_lower_arm |
+    | 12 | left_upper_arm |
+    | 13 | left_lower_arm |
+
+    The joints are:
+
+    | id (for `v2`,`v3`,`v4`) | joint |
```
Comment (pseudo-rnd-thoughts) [inline gymnasium/envs/mujoco/humanoidstandup_v4.py:153]: Fix columns

**Review by pseudo-rnd-thoughts — CHANGES_REQUESTED** [review]

File: gymnasium/envs/mujoco/walker2d_v4.py (hunk 1 above)
Comment (Kallinteris-Andreas) [reply gymnasium/envs/mujoco/walker2d_v4.py]: i dunno it is fine now

**Review by pseudo-rnd-thoughts — APPROVED** [review]

**Pull request merged**